src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/DisplayWfs/DisplayWfs.xaml.cs

```diff
@@ -0,0 +1,111 @@
+        private const string ServiceUrl = "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities";
+        private const string LayerName = "Seattle_Downtown_Features:Buildings";
+
+        public DisplayWfs()
+        {
+            InitializeComponent();
+            Initialize();
+        }
+
+        private async void Initialize()
+        {
+            // Create the map with topographic basemap.
+            MyMapView.Map = new Map(Basemap.CreateTopographic());
+
+            try
+            {
+                // Create the feature table from URI and layer name.
+                _featureTable = new WfsFeatureTable(new Uri(ServiceUrl), LayerName);
+
+                // Set the feature request mode to manual - only manual is supported at v100.5.
+                _featureTable.FeatureRequestMode = FeatureRequestMode.ManualCache;
+
+                // Load the table.
+                await _featureTable.LoadAsync();
+
+                // Create a feature layer to visualize the WFS features.
+                FeatureLayer wfsFeatureLayer = new FeatureLayer(_featureTable);
+
+                // Apply a renderer.
+                wfsFeatureLayer.Renderer = new SimpleRenderer(new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, Color.Red, 3));
+
+                // Add the layer to the map.
+                MyMapView.Map.OperationalLayers.Add(wfsFeatureLayer);
+
+                // Use the navigation completed event to populate the table with the features needed for the current extent.
+                MyMapView.NavigationCompleted += MapView_NavigationCompleted;
+
+                // Zoom to a small area within the dataset by default.
+                MapPoint topLeft = new MapPoint(-122.341581, 47.617207, SpatialReferences.Wgs84);
+                MapPoint bottomRight = new MapPoint(-122.332662, 47.613758, SpatialReferences.Wgs84);
+                await MyMapView.SetViewpointGeometryAsync(new Envelope(topLeft, bottomRight));
+            }
+            catch (Exception e)
+            {
+                MessageBox.Show(e.ToString(), "Couldn't load sample.");
+                Debug.WriteLine(e);
+            }
+        }
+
+        private async void MapView_NavigationCompleted(object sender, EventArgs e)
+        {
+            // Show the loading bar.
+            LoadingProgressbar.Visibility = Visibility.Visible;
+
+            // Get the current extent.
+            Envelope currentExtent = MyMapView.VisibleArea.Extent;
+
+            // Create a query based on the current visible extent.
+            QueryParameters visibleExtentQuery = new QueryParameters();
+            visibleExtentQuery.Geometry = currentExtent;
+            visibleExtentQuery.SpatialRelationship = SpatialRelationship.Intersects;
+
+            try
+            {
+                // Populate the table with the query, leaving existing table entries intact.
```

Comment:
More explicit comments:
// Populate the WFS feature table with the query, leaving existing table entries intact. Since we are not interested in using any attribute data, set the outFields parameter to null.